demo/src/renderDemo.js

```diff
@@ -54,26 +54,41 @@ function getComponent(name) {
   return registry[name];
 }
 
+function Demo() {
+  const fixedHeight = 600;
+  const [isExpanded, setIsExpanded] = useState(true);
+  function toggleIsExpanded() {
+    setIsExpanded(prev => !prev);
+  }
+  return (
+    <div style={(isExpanded ? { height: `${fixedHeight}px`, width: `${fixedHeight}px` } : {})}>
+      <button onClick={toggleIsExpanded}>Toggle Expanded Grid</button>
+      <VitessceGrid
+        layout={layout}
+        getComponent={getComponent}
+        draggableHandle={`.${handleClass}`}
+        padding={50}
+        height={isExpanded}
+        rowHeight={(isExpanded ? fixedHeight/4 : undefined)} /* If undefined, will fill window height. */
```

Comment:
why divided by 4?

Reply:
there are four "rows" in the layout in `demo/src/layout.json`